### DISCUSSION TITLE
Global Finalist Selection Process

### DISCUSSION THREAD

**Discussion by Aditya-Jha2002** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#81] [General]
I wanted to understand the top 60 global finalist selection process.
I got a ranking of 20 in the second challenge and am ranked 13 in the APAC Region.
Since there are only 3 candidates from the America region, is it a possibility that the rest spots be filled from other regions?,
Or will there be less than 60 finalists?

**Comment by EY-glowglowglow** [comment]
Hi there,
In this case there will be less than 60 global finalists. Thank you for your participation in the challenge, and I hope you've had fun and took away some learnings from it. :)

**Reply by firasam** [reply]
for the global finalist is it going to be selected by selecting the top 10 of each region or the overall top 30 of the leaderboard?

**Reply by EY-glowglowglow** [reply]
Hi @firasam, it will be top 10 for each region.

**Reply by firasam** [reply]
@EY-glowglowglow for the global finalist it is supposed to be top 10 for each region right? why is on the announcement global finalist only consist of top 12 from overall leaderboard?

**Comment by Aditya-Jha2002** [comment]
Oh Ok, yeah I had a lot of fun, thank you :)

**Comment by patricktsanght** [comment]
Will the finalist models be published? This way we can learn from the best :)
